src/modelldcatnotordf/informationmodel.py

```diff
@@ -90,9 +90,22 @@ def to_rdf(
         """
         return self._to_graph().serialize(format=format, encoding=encoding)
 
+    def unionof(self: Resource, graph: Graph) -> Graph:
```

Comment:
Det var mye kode for lite som skjer :P

Kunne du ikke like så greit droppet `_publisher_to_graph` og `unionof`?
Og i stedet hatt `_to_graph` som dette:
```
def _to_graph(self: Resource) -> Graph:
        super(InformationModel, self)._to_graph()
        self._g.add((URIRef(self.identifier), RDF.type, self._type))
        self._g = self._g + self.publisher.to_graph()
        return self._g
```

Reply:
kanskje til og med `self._g += self.publisher.to_graph()`?

Reply:
Tanken med dette var at koden skulle avspeile litt "forretningsomårdet" og dermed hva vi gjør sånn logikk- og semantisk web-messig. ;-)

Reply:
logikken er jo der :P

Men ok, hva med å droppe `publisher_to_graph` og gjør `self._g = self.unionof(self.publisher.to_graph())` der i stedet?

Reply:
Men hvis du 'må' ha så bloaty kode så kan du i det minste endre navnet på `_publisher_to_graph` til `_add_publisher_to_graph`, slik at navnet faktisk viser hva som skjer

Reply:
Ok, det er jeg enig i. Det er en gylden middelvei mellom å ha ikke-bloaty kode på den ene siden og at koden skal avspeile det funksjonelle området på den andre siden.. ;-)

Reply:
Det siste der var visst ikke likefrem å få til. Å endre _publisher_to_graph til _add_publisher_to_graph går ikke uten videre fordi det ser ut som om metoden må overlastes siden information model er en dcat:Resource og publisher hører til her.

Reply:
Ok,greit nok